-------------------------------------------------------------------------------
Test set: org.apache.hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestroy
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 67.529 s <<< FAILURE! - in org.apache.hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestroy
testPipelineCreationOnNodeRestart(org.apache.hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestroy)  Time elapsed: 22.264 s  <<< FAILURE!
java.lang.AssertionError: expected:<FAILED_TO_FIND_SUITABLE_NODE> but was:<FAILED_TO_FIND_HEALTHY_NODES>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.apache.hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestroy.testPipelineCreationOnNodeRestart(TestRatisPipelineCreateAndDestroy.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)

